sourmash/signature_json.py

```diff
@@ -236,15 +235,8 @@ def load_signatures_json(data, ksize=None, ignore_md5sum=True, ijson=ijson):
         notify('\r...sig loading {:,}', n, flush=True)
 
 
-def save_signatures_json(siglist, fp=None, indent=None, sort_keys=True):
-    """ Save multiple signatures into a JSON string (or into file handle 'fp')
-    - siglist: sequence of SourmashSignature objects
-    - fp:
-    - indent: indentation spaces (an integer) or if None no indentation
-    - sort_keys: sort the keys in mappings before writting to JSON
-    """
-    from .signature import SIGNATURE_VERSION
-
+def get_top_records(siglist):
+    notify("in get_top_records", end="\r")
```

Comment:
see comment above, re making this into either `debug` or something that the user can see and understand.